examples/weather_skills/README.md

```diff
@@ -53,10 +62,16 @@ ledger_apis:
     addr: alpha.fetch-ai.com
     port: 80
 ```
+- Generate the private key for the weather client:
```

Comment:
there must be a better way to generate private keys... `aea generate_keys [fetch|ethereum|all]`?

Reply:
Yes, great point!

Reply:
We had a discussion on that. Why don't we generate the keys on the create. Don't remember the reason why we didn't. Moreover, is it possible to support more than one private/public key per agent ?